### PR TITLE
fix: add cornichons fins

### PR DESCRIPTION
**Pull request by teolemon**
### What
fix: add cornichons fins

### CODE REVIEW

**Review by stephanegigandet — COMMENTED** [review]
For new categories, please always add an English name as the main entry to avoid duplicates in other languages

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -95233,6 +95233,17 @@ fi:hienoja suolakurkkuja
...
+# TODO: check if Cornichons très fins, mini-cornichons are legally the same
+
+<en:Gherkins
+en:Fine gherkins
+de:Feine Cornichons
+fr:Cornichons fins
+nl:Fijne augurken
+
+<en:Gherkins
+fr:Cornichons mi-fins
```
Comment (benbenben2) [inline taxonomies/categories.txt:95245]:
```suggestion
en:semi-fine gherkins
fr:Cornichons mi-fins
```

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -95244,6 +95255,24 @@ ciqual_food_code:en:11097
 ciqual_food_name:en:Gherkin, in sweet and sour sauce
 ciqual_food_name:fr:Cornichon, aigre-doux
 
+<en:Gherkins
+fr:Cornichons à l'aneth
```
Comment (benbenben2) [inline taxonomies/categories.txt:95259]:
```suggestion
en:dill pickles
fr:Cornichons à l'aneth
```

File: taxonomies/categories.txt [hunk 3]
```diff
@@ -95244,6 +95255,24 @@ ciqual_food_code:en:11097
...
+
+<en:Gherkins
+fr:Cornichons à la polonaise
```
Comment (benbenben2) [inline taxonomies/categories.txt:95262]:
```suggestion
en:gherkins with dill
fr:Cornichons à la polonaise
```

File: taxonomies/categories.txt [hunk 4]
```diff
@@ -95244,6 +95255,24 @@ ciqual_food_code:en:11097
...
+
+<en:Gherkins
+fr:Cornichons à la russe
```
Comment (benbenben2) [inline taxonomies/categories.txt:95265]:
```suggestion
en:Russian pickles
fr:Cornichons à la russe
```

File: taxonomies/categories.txt [hunk 5]
```diff
@@ -95244,6 +95255,24 @@ ciqual_food_code:en:11097
...
+
+<en:Gherkins
+fr:Cornichons en rondelles, Rondelles de cornichons, Tranches de cornichons, Cornichons en tranches, Cornichons tranchés, Cornichons en lamelles
```
Comment (benbenben2) [inline taxonomies/categories.txt:95268]:
```suggestion
en:Gherkins slices
fr:Cornichons en rondelles, Rondelles de cornichons, Tranches de cornichons, Cornichons en tranches, Cornichons tranchés, Cornichons en lamelles
```

File: taxonomies/categories.txt [hunk 6]
```diff
@@ -95244,6 +95255,24 @@ ciqual_food_code:en:11097
...
+
+<en:Gherkins
+fr:Cornichons au piment d'espelette
```
Comment (benbenben2) [inline taxonomies/categories.txt:95271]:
```suggestion
en:Gherkins with espelette pepper
fr:Cornichons au piment d'espelette
```

File: taxonomies/categories.txt [hunk 7]
```diff
@@ -95244,6 +95255,24 @@ ciqual_food_code:en:11097
...
+
+<en:Gherkins
+fr:Cornichons aux oignons
```
Comment (benbenben2) [inline taxonomies/categories.txt:95274]:
```suggestion
en:Gherkins with onions
fr:Cornichons aux oignons
```

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Pull request merged**